var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -364,3 +365,21 @@ def command(self):
         else:
             msg = 'Unable to locate {0} command in {1}'
             raise RuntimeError(msg.format(self.spec.name, self.prefix.bin))
+
+    def test(self):
+        perl = which('perl')
```

Comment:
These tests won't work for development releases of Perl. Development releases don't come with a `perl` executable, only `perlX.Y.Z`. You should use `self.spec['perl'].command` like you for for Python.

Reply:
This test was supposed to have been converted -- commit must've gotten lost -- to the newer, `run_tests` approach, for more consistent tracking.

But I'll make the change here since it shows an alternate way of handling the test.

Reply:
After doing that I and seeing the results I decided to change it to use the preferred approach.